src/components/general/SearchableDropdown/index.tsx

```diff
@@ -186,7 +186,7 @@ const SearchableDropdown = ({
         const overlayContainer = useOverlayContainer();
         const handleBlur = useCallback(
             (e: FocusEvent<HTMLInputElement>) => {
-                if (overlayContainer.contains(e.relatedTarget as HTMLElement)) return;
+                if (!overlayContainer.contains(e.relatedTarget as HTMLElement)) return;
```

Comment:
Not sure about this one. I negated the if-statement because `overlayContainer` will always contain `e.relatedTarget`. 
After negating it, you can use Tab from one SearchableDropdown to another, and the former will close.
overlayContainer (which comes from  `const overlayContainer = useOverlayContainer();`) should maybe be replaced with something else